pagopa/io-app
# fix: [IOBP-69] Fix long IDPay initiatives names in wallet cards

## Short description
This PR fixes the visualization of initiatives cards with long names in wallet.

| Before | After |
| --- | --- |
|  <img src="https://github.com/pagopa/io-app/assets/6160324/b60e4f09-d47c-41f5-aee9-97174d807cee" width="200" /> | <img src="https://github.com/pagopa/io-app/assets/6160324/1b9540b6-aeae-428b-a13d-aa4dc04bffe8" width="200" />  |

## List of changes proposed in this pull request
- Fixed styles in `IDPayCardPreviewComponent`

## How to test
With the `io-dev-server`, change the IDPay wallet payload to show initiatives with very long names. Check that cards in wallet shows the truncated names.
